src/openrct2/ride/RideData.cpp

```diff
@@ -783,117 +783,6 @@ const uint8_t rideBonusValue[RIDE_TYPE_COUNT] = {
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0d Bobsleigh Coaster
-    { RIDE_COMPONENT_TYPE_CABIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0e Observation Tower
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0f Looping Roller Coaster
-    { RIDE_COMPONENT_TYPE_BOAT,         RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 10 Dinghy Slide
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 11 Mine Train Coaster
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 12 Chairlift
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 13 Corkscrew Roller Coaster
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 14 Maze
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 15 Spiral Slide
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 16 Go Karts
-    { RIDE_COMPONENT_TYPE_BOAT,         RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 17 Log Flume
-    { RIDE_COMPONENT_TYPE_BOAT,         RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 18 River Rapids
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 19 Dodgems
-    { RIDE_COMPONENT_TYPE_SHIP,         RIDE_COMPONENT_TYPE_STRUCTURE,          RIDE_COMPONENT_TYPE_STATION }, // 1a Pirate Ship
-    { RIDE_COMPONENT_TYPE_SHIP,         RIDE_COMPONENT_TYPE_STRUCTURE,          RIDE_COMPONENT_TYPE_STATION }, // 1b Swinging Inverter Ship
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 1c Food Stall
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 1d (none)
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 1e Drink Stall
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 1f (none)
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 20 Shop (all types)
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_STRUCTURE,          RIDE_COMPONENT_TYPE_STATION }, // 21 Merry Go Round
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 22 Balloon Stall (maybe)
```

Comment:
Where did this go? I didn't find a Baloon File.

Reply:
It's because the comment is wrong. 0x22 is a non-existing ride.